src/pagination/pagination.spec.ts

```diff
@@ -641,6 +641,18 @@ describe('ngb-pagination', () => {
            expect(buttons[i]).toHaveCssClass('disabled');
          }
        }));
+
+    it('should set tabindex for links correctly for disabled state', fakeAsync(() => {
```

Comment:
This test doesn't need to be `fakeAsync` and `tick()` is also not necessary.
The `should set classes correctly for disabled state` you used as a base is also wrong and doesn't need them.

Do you mind fixing both please, while you're at it ?